debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -693,6 +710,11 @@ public void shouldReceiveChangesForUpdatesWithPKChanges() throws Exception {
         assertEquals(topicName, deleteRecord.topic());
         VerifyRecord.isValidDelete(deleteRecord, PK_FIELD, 1);
 
+        assertEquals(1, deleteRecord.headers().size()); // to be removed/updated once we set additional headers
```

Comment:
I'm a bit confused by these. Are these TODO markers for later (after this PR), or are you planning to address them as part of this PR?

Reply:
no, they are there for future me (and you). When we decide to add more headers in the future, this comparison will fail because total header count will be more than 1.

Reply:
Ok, I see. I'd suggest to remove those comments then, as they are confusing IMO. It's expected that existing tests needs adjustment after changes down the road. Right now it sounds a bit as if there's a loose end.

Reply:
Remove comments or remove the these asserts, or rephrase? 

Don't you think that people working on adding headers in the future will get failing tests and need a lot of time to figure out that they should just update or remove that comparison to 1? I put it because I thought it could annoy future me? But maybe I am too careful here?

Reply:
Generally speaking, I think if someone adds a feature introducing a new header down the road, they'd quite quickly see why this assertion is failing. But taking a step back, let's just remove the assertion. You assert on the actual header value on the next line anyways, and that's what matters for this test here. It does't really need to be concerned with other headers being present.